src/routes/controllers/utils/helper.js

```diff
@@ -0,0 +1,45 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+
+const secret = process.env.SECRET_KEY;
+
+const Help = {
+  /**
+   * Hash Password Method
+   * @param {string} password
+   * @returns {string} returns hashed password
+   */
+  hashPassword(password) {
```

Comment:
We can do this in the model using getters and setters, see http://docs.sequelizejs.com/manual/models-definition.html#getters--amp--setters for more info on this.

Use the setters to hash the password before it is saved in the database.

Reply:
Okay. Noted